src/NetworkManagement2/packages.config

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Hyak.Common" version="1.0.2" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common" version="2.1.0" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common.Dependencies" version="1.0.0" targetFramework="portable-net45+sl50+win+wp80" />
+  <package id="Microsoft.Azure.Management.Resources" version="2.12.0-preview" targetFramework="net40" />
```

Comment:
@Nilambari We should not be taking dependency on Resources. I check in -pr Build branch and i dont find it there. it should not required for ignite as well. Can we remove this dependency please?